opm/autodiff/StandardWell_impl.hpp

```diff
@@ -634,6 +634,71 @@ namespace Opm
                     well_state.perfPhaseRates()[(first_perf_ + perf) * np + ebosCompIdxToFlowCompIdx(componentIdx)] = cq_s[componentIdx].value();
                 }
             }
+            if (GET_PROP_VALUE(TypeTag, EnableEnergy)) {
```

Comment:
Same with `has_energy`. Since it is already defined in the well class, maybe it's better to use that one within the class for consistence?